tests/integration-tests/tests/multiple_nics/test_multiple_nics/test_multiple_nics/pcluster.config.yaml

```diff
@@ -1,7 +1,7 @@
 Image:
   Os: {{ os }}
 HeadNode:
-  InstanceType: {{ instance }}
+  InstanceType: r6in.metal
```

Comment:
let's add a comment here (and in the comment message too) to explain that to verify multi-nics capabilities we can use any instance type with multiple network card and the link to official doc with the list.